### DISCUSSION TITLE
Unhandled Runtime Error: "Failed to Fetch Drinks" in Next.js

### DISCUSSION THREAD

**Discussion by MarwanEA** [community/community#135704] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm encountering an "Unhandled Runtime Error: Failed to fetch drinks" in my Next.js application. The error occurs when trying to fetch data from an API endpoint using fetch. Below is a screenshot of the error.

Question:

What could be causing this issue, and how can I resolve it?

![c96c6864-5195-4761-8bb2-cdca30cfc618](https://github.com/user-attachments/assets/126ef8ef-426a-4999-869a-d4d8a7c9148f)

**Comment by Achraftraf** [comment, marked as answer]
You can create it as a page within the same folder, for example.

<img width="911" alt="image" src="https://github.com/user-attachments/assets/5134887a-232c-4d8f-a070-cbd983e7b074">

the result :
<img width="918" alt="image" src="https://github.com/user-attachments/assets/952c8103-319e-4c6b-aeed-334d3d003ef8">
